js/settings.js

```diff
@@ -27,4 +27,19 @@ $(document).ready(function () {
 			}
 		}, false);
 	}
+
+	var testToken = Math.random().toString();
+	$.ajax({
+		type: 'POST',
+		url: OC.generateUrl('apps/oauth2/test'),
+		headers: {
+			'Authorization': testToken
```

Comment:
Bearer is Missing

Reply:
true, but the content doesn't matter here, I just check whether the header is passed to PHP

Reply:
Fixed

Reply:
Apache is explicitly removing is the auth header starts with Bearer :man_shrugging:

Reply:
hm. I did tested it manually before push and for me any kind of Authorization header was removed